python/cudf/io/csv.py

```diff
@@ -143,12 +155,13 @@ def read_csv(filepath, lineterminator='\n',
 
 
 def read_csv_strings(filepath, lineterminator='\n',
+                     quotechar='\0', quoting=True, doublequote=True,
                      delimiter=',', sep=None, delim_whitespace=False,
                      skipinitialspace=False, names=None, dtype=None,
                      skipfooter=0, skiprows=0, dayfirst=False):
 
     import nvstrings
-    from .series import Series
+    from cudf.dataframe.series import Series
 
     """
     **Experimental**: This function provided only as an alpha way of providing
```

Comment:
Looks like some previous commits got lost in the force push. I'll add a commit with the updated example docstring.